multiplex.json

```diff
@@ -9,7 +9,7 @@
     "validations": {
 	"pci_device_list": {
 	    "description": "1 or more (comma separated) device pairs",
-	    "args": [ "devices", "active-devices", "testpmd-devopt" ],
+	    "args": [ "devices", "active-devices" ],
```

Comment:
is "active-devices" still a thing ?
I see devices only at https://github.com/perftool-incubator/bench-trafficgen/blob/203e02e5840d87a3d1086a3269a3d14e70d63029/trafficgen-server-start#L45

Reply:
`active-devices` only applies to TRex (client/infra role)